notebooks/examples/python/QuickstartNotebook.py

```diff
@@ -1,14 +1,49 @@
 # Databricks notebook source
 # MAGIC %md
 # MAGIC ## Enable Mosaic in the notebook
-# MAGIC To get started, you'll need to attach the JAR to your cluster and import instances as in the cell below.
+# MAGIC To get started, you'll need to attach the wheel to your cluster and import instances as in the cell below.
 
 # COMMAND ----------
 
 from pyspark.sql.functions import *
-from mosaic import enable_mosaic
-enable_mosaic(spark, dbutils)
-from mosaic import *
+import mosaic as mos
+mos.enable_mosaic(spark, dbutils)
+
+# COMMAND ----------
+
+user_name = dbutils.notebook.entry_point.getDbutils().notebook().getContext().userName().get()
```

Comment:
@edurdevic love this! this is what I should have done in the first place :)

Reply:
:)